### PR TITLE
Python3

### PR DESCRIPTION
**Pull request by christophwelling**
Implements changes that are necessary for NuRadioReco to run on python3.
While the example and test reconstructions run without errors and the travis unit test passes, we should set up more tests before merging this.
The changes to the ARIANNA reader for py3 are still on a separate branch, so we would first have to merge that one.

Also, if you want to try this, make sure your radiotools are on the python3 branch.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,9 +11,9 @@ before_script:
   - unzip /tmp/radiotools.zip
   - mv radiotools-master radiotools
   - export PYTHONPATH=$PWD/radiotools
-  - wget https://github.com/nu-radio/NuRadioMC/archive/master.zip -O /tmp/NuRadioMC.zip
+  - wget https://github.com/nu-radio/NuRadioMC/archive/python3.zip -O /tmp/NuRadioMC.zip
   - unzip /tmp/NuRadioMC.zip
-  - mv NuRadioMC-master NuRadioMC
```
Comment (anelles) [inline .travis.yml]: We should rememer to change this right before submission/after NuRadioMC is merged.

File: NuRadioReco/modules/io/eventWriter.py [hunk 2]
```diff
@@ -29,8 +29,10 @@ class eventWriter:
 
     def __write_fout_header(self):
         b = bytearray()
-        b.extend(b'{:06x}'.format(VERSION))
-        b.extend(b'{:06x}'.format(VERSION_MINOR))
+        b.extend(VERSION.to_bytes(6, 'little'))
+        b.extend(VERSION_MINOR.to_bytes(6, 'little'))
+        #b.extend(b'{:06x}'.format(VERSION))
```
Comment (anelles) [inline NuRadioReco/modules/io/eventWriter.py]: Clean-up?

Comment (christophwelling) [reply NuRadioReco/modules/io/eventWriter.py]: Done

**Comment by anelles** [comment]
Let's give @cg-laser also a chance, but after he did excessive testing, I think we are fine.

**Comment by cg-laser** [comment]
I'm traveling today, I'll have a look tomorrow (Friday)

--

**Comment by cg-laser** [comment]
almost forgot, we're now at version 1.0.0 :-)

**Comment by christophwelling** [comment]
> I'll let the honor to @christophwelling to merge this important step! I'll create a first python3 release next Monday

Should we already make a release? There are a lot of changes coming up that are not backward-compatible, so that release would quickly be outdated

**Pull request merged**